db/migrate/20230515082603_add_new_fields_to_english_language_providers.rb

```diff
@@ -0,0 +1,8 @@
+class AddNewFieldsToEnglishLanguageProviders < ActiveRecord::Migration[7.0]
+  def change
+    change_table :english_language_providers, bulk: true do |t|
+      t.string :provider_url, null: false, default: ""
```

Comment:
```suggestion
      t.string :url, null: false, default: ""
```

I don't think we need the `provider_` prefix as we're already within the "English language provider" model, so we're saying it's the `url` of the `EnglishLanguageProvider`.